packages/client/src/__tests__/buffer-small.test.ts

```diff
@@ -0,0 +1,6197 @@
+/* eslint-disable no-global-assign */
+/* eslint-disable @typescript-eslint/no-unused-vars */
+/* eslint-disable @typescript-eslint/no-loss-of-precision */
+
+import assert from 'assert'
+
+import { Buffer } from '../../../../helpers/compile/plugins/fill-plugin/fillers/buffer-small'
+
+const describeIf = (condition: boolean) => (condition ? describe : describe.skip)
+const testIf = (condition: boolean) => (condition ? test : test.skip)
+
+describe = describeIf(!process.version.startsWith('v16.'))
+test = testIf(!process.version.startsWith('v16.'))
+
+describe('Buffer isBuffer (static)', () => {
+  test('should return true if the object is a Buffer', () => {
+    const buf = Buffer.from('Hello, World!')
+    expect(Buffer.isBuffer(buf)).toBe(true)
+  })
+
+  test('should return false if the object is not a Buffer', () => {
+    const notBuf = 'Hello, World!'
+    expect(Buffer.isBuffer(notBuf)).toBe(false)
+  })
+})
+
+describe('Buffer from (static)', () => {
+  test('should create a new Buffer from a string', () => {
+    const buf = Buffer.from('Hello, World!')
+    expect(buf.toString()).toEqual('Hello, World!')
+  })
+
+  test('should create a new Buffer from an array', () => {
+    const buf = Buffer.from([0x48, 0x65, 0x6c, 0x6c, 0x6f])
+    expect(buf.toString()).toEqual('Hello')
+  })
+
+  test('should create a new Buffer from a Buffer', () => {
+    const buf1 = Buffer.from('Hello, World!')
+    const buf2 = Buffer.from(buf1)
+    expect(buf2.toString()).toEqual('Hello, World!')
+  })
+
+  test('should create a new Buffer from an ArrayBuffer', () => {
+    const arr = new Uint16Array([0x48, 0x65, 0x6c, 0x6c, 0x6f])
+    const buf = Buffer.from(arr.buffer)
+    expect(buf.toString()).toEqual('H\u0000e\u0000l\u0000l\u0000o\u0000')
+  })
+
+  test('should throw an error if input is invalid', () => {
+    // expect(() => Buffer.from(null)).toThrow() ❌
```

Comment:
Does not seem like adding validation here would have that big of an impact on the size.